scripts/clusterCreation/deployASB.sh

```diff
@@ -0,0 +1,609 @@
+#!/bin/bash
+
+function collectInputParameters()
+{
+  echo $1
+  if [ -z $1 ]; then
+    echo "Please provide the Cluster Admin ID when calling this script" 1>&2
+    echo "This command can be retrieved by running ./clusterCreation/getClusterAdminIDforDeployment.sh from your local machine (not CodeSpaces)" 1>&2
+    exit 1
+  else 
+    export ASB_CLUSTER_ADMIN_ID=$1
+
+    export ASB_SCRIPT_STEP=setDeploymentName
+
+      # Save environment variables
+    ./saveenv.sh -y
+
+    # Invoke Next Step In Setup
+    $ASB_SCRIPT_STEP
+  fi
+}
+
+function setDeploymentName()
+{
+  function isValidDeploymentName() {
+    [[ $1 =~ ^[a-z]([a-z]|\d){2,8}$ ]]
```

Comment:
This does not seem to allow numbers in the name. I tried using `ak01` but could not get past this validation when running the script.

Reply:
fixed this by using [[:digit:]] instead of \d